subsys/bootloader/Kconfig

```diff
@@ -226,7 +226,7 @@ config NRF53_UPGRADE_NETWORK_CORE
 	bool "Support updating nRF53 Network Core application"
 	default y
 	depends on SOC_NRF5340_CPUAPP
-	depends on BOOTLOADER_MCUBOOT
+	depends on BOOTLOADER_MCUBOOT || MCUBOOT
```

Comment:
Indeed that was incorrect. This change is no longer needed though because of https://github.com/nrfconnect/sdk-mcuboot/commit/22da38d2f0bd10e678cc8b061adbdcd3c964076a

Reply:
> This change is no longer needed though

and yet it still remains here 🤔

Reply:
I'm adding the nrf53 multi image update commit first (y)